.github/ISSUE_TEMPLATE/patch_release.md

```diff
@@ -33,6 +33,15 @@ according to the declared schedule, you should have:
 - [ ] Cut [UXP][uxp] `vX.Y.Z-up.K` release from the `release-X.Y` branch by:
   - [ ] Running the [Tag workflow][tag-uxp] on the `release-vX.Y` branch with the proper release version, `vX.Y.Z-up.K`. Use `Release vX.Y.Z-up.K` as message (FYI: the format suggested is only for consistency, there is no actual dependency on it).
   - [ ] Running the [CI workflow][ci-uxp] on the `release-vX.Y` branch to build and publish the latest tagged artifacts.
+  - [ ] Verify that the tagged build version exists on the [releases.upbound.io](https://releases.upbound.io/universal-crossplane/) `build` channel, e.g. `build/release-X.Y/vX.Y.Z-up.K/...`
+- [ ] Verify the produced helm chart available in the `build` channnel at `build/release-X.Y/vX.Y.Z-up.K/charts` by doing some sanity checks:
+  - [ ] Installs on a cluster properly with `helm -n upbound-system install universal-crossplane <path-to-chart.tgz> --create-namespace`.
```

Comment:
Helm allows directly using an url to the tgz, what do you think about directly showing the whole example command as below?
```suggestion
  - [ ] Installs on a cluster properly with `helm -n upbound-system install universal-crossplane <path-to-chart.tgz> --create-namespace`, e.g. `helm -n upbound-system install universal-crossplane https://releases.upbound.io/universal-crossplane/build/release-X.Y/vX.Y.Z-up.K/charts/universal-crossplane-X.Y.Z-up.K.tgz --create-namespace`.
```